### PR TITLE
OP-312 Telemetry Module

### PR DESCRIPTION
**Pull request by mwithi**
See OP-312.

### CODE REVIEW

File: rsc/telemetry.properties [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# number of second to check if it should send a message
```
Comment (dbmalkovsky) [inline rsc/telemetry.properties:1]:
```suggestion
# number of seconds to check if it should send a message

```

File: rsc/telemetry.properties [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# number of second to check if it should send a message
+telemetry.daemon.thread.loop.seconds=14400
```
Comment (dbmalkovsky) [inline rsc/telemetry.properties:2]: Do you mean to mix line endings?

**Review by dbmalkovsky — APPROVED** [review]
LGTM with a few comments

File: rsc/telemetry.properties (hunk 2 above)
Comment (mwithi) [reply rsc/telemetry.properties:3]: strange! let me fix that

**Pull request merged**